### DISCUSSION TITLE
Request for option to print/return only the hash value

### DISCUSSION THREAD

**Discussion by JavascriptCockup** [rhash/RHash#217] [Ideas]
As title, would like an even simpler version of the --simple output format option, where only the actual hash value is returned - no filename or other information.

**Comment by greisch** [comment]
Already implemented.
use option --printf=<format>

example:
\# rhash --printf="%c\n" filename.bin
a94a868c

%c is for crc32 ; %C for crc32 in upercase
check manual for all checksum alternatives
\# man rhash

**Comment by rhash** [comment]
Implemented in [RHash v1.4.5](https://github.com/rhash/RHash/releases/tag/v1.4.5), see FR #259.

Usage example:
```sh
$ rhash --one-hash --md5 --message "abc"
900150983cd24fb0d6963f7d28e17f72
```
